docs/module_federation/nextjs_plugin.md

```diff
@@ -97,8 +97,10 @@ yarn add @module-federation/nextjs-mf
 
 Create a new <b>header</b> application which will be our remote module, we can do this manually or by using the Nx CLI.
 
+Make sure to select <b>No</b> when prompted to use the App Router.
+
 ```bash
-nx g @nx/next:app header
+nx g @nrwl/next:app header
```

Comment:
We could just specify the --no-appDir flag as part of the command we show to use, so the command would be

`nx g @nx/next:app header --no-appDir`

This would ensure that someone doesn't set app router to true by accident. 

Also it should be 'nx' not 'nrwl', that part was already correct. nrwl is what it used to be.

Reply:
Updated